File: sct_source/SourceCode/API_SoCongThuong/Models/InterCommerceModel.cs
namespace API_SoCongThuong.Models
{
    public class InterCommerceModel
    {
        public Guid InternationalCommerceId { get; set; }
        public Guid InternationalCommerceName { get; set; }
        public string InvestorName { get; set; } = "";
        public string Address { get; set; } = "";
        public string LicensingActivity { get; set; } = "";
        public string Representative { get; set; } = "";
        public string BusinessNameVi { get; set; } = "";
        public string PhoneNumber { get; set; } = "";
        public string? TenCoSoBanLe { get; set; }
        public string? DiaChiCoSoBanLe { get; set; }
        public Guid? LoaiHinhCoSo { get; set; }
        public string? GiayPhepKinhDoanh { get; set; }
        public DateTime? NgayCapGiayPhepKinhDoanh { get; set; }
        public string? GiayPhepBanLe { get; set; }
        public DateTime? NgayCapGiayPhepBanLe { get; set; }
        public DateTime? NgayHetHanGiayPhepBanLe { get; set; }
        public decimal DienTichSuDung { get; set; }
        public decimal DienTichSan { get; set; }
        public decimal DienTichBanHang { get; set; }
        public decimal DienTichKinhDoanh { get; set; }
        public string? GhiChu { get; set; }
        public bool IsDel { get; set; }
        public string TenLoaiHinhCoSo { get; set; } = "";
    }
    public class removeListInterCommerceItems
    {
        public List<Guid> InternationalCommerceIds { get; set; }
    }
}

File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/inter-commerce-page/_models/inter-commerce.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface InterCommerceModel extends BaseModel {
  // CigaretteBusinessId : string;
  InternationalCommerceId : string;
  internationalCommerceName : string;
  investorName : string;
  licensingActivity : string;
  address : string;
  diaChiDoanhNghiep: string;
  giayDangKyKinhDoanh: string;
  ngayCapPhep: any;
  nguoiDaiDien: string;
  soDienThoai: string;
  tenCoSoBanLe: string;
  diaChiCoSoBanLe: string;
  giayPhepKinhDoanh: string;
  ngayCapGiayPhepKinhDoanh: any;
  giayPhepBanLe: string;
  ngayCapGiayPhepBanLe: any;
  ngayHetHanGiayPhepBanLe: any;
  dienTichSuDung: number;
  dienTichSan: number;
  dienTichBanHang: number;
  dienTichKinhDoanh: number;
  ghiChu: string;
  loaiHinhCoSo: string;
}
